library/api.inc

```diff
@@ -1,6 +1,6 @@
 <?php
 //our api for 3rd party developers
-include_once("../../globals.php");
+include_once(dirname(__FILE__) . "../interface/globals.php");
```

Comment:
I think you'll need a slash at the beginning:
``````
include_once(dirname(__FILE__) . "/../interface/globals.php");